src/stock/classifying.js

```diff
@@ -278,8 +278,52 @@ export const getSZ50 = () => {
   const url = sz50Url();
 
   return fetch(url)
-  .then(checkStatus)
-  .then(res => res.json())
-  .then(json => ({ data: arrayObjectMapping(json[0].fields, json[0].items) }))
-  .catch(error => ({ error }));
+        .then(checkStatus)
+        .then(res => res.json())
+        .then(json => ({ data: arrayObjectMapping(json[0].fields, json[0].items) }))
+        .catch(error => ({ error }));
 };
+
+/**
+ * 获取指定年月限售解禁股数据
+ * 返回数据格式: 数组
+ * [
+ *   {
+ *     symbol: 股票代码
+ *     name: 股票名称
+ *     date: 解除限售日期
+ *     percent: 占总股本比例
+ *     count: 数量（万股）
+ *     close: 最新收盘价（元）
+ *     curTotalValue: 当前市值（亿元）
+ *   }
+ * ]
+ * @param year
+ * @param month
+ * @returns {Promise.<TResult>}
+ */
+export const getXSGData = (year, month) => {
+  const url = xsgUrl(year, month);
+  const mapData = data => {
+    let arr = JSON.parse(data.substring(1, data.length - 1));
+    arr = arr.map(item => {
+      const itemData = item.split(',');
+      return {
+        symbol: itemData[1],
+        name: itemData[3],
+        date: itemData[4],
+        percent: itemData[6],
+        count: (itemData[5] / 10000).toFixed(2),
+        close: itemData[7],
+        curTotalValue: (itemData[8] / 100000000).toFixed(4),
+      };
+    });
+    return arr;
+  };
+  return fetch(url)
+        .then(checkStatus)
+        .then(res => res.text().then(result => result))
```

Comment:
oh no,It should be deleted,I have forgot it.Thank you pointed out. I'll fixed it.